src/Symfony/Bridge/PhpUnit/Legacy/ConstraintTraitForV9.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Legacy;
+
+/**
+ * @internal
+ */
+trait ConstraintTraitForV9
+{
+    use ConstraintLogicTrait;
+
+    public function evaluate($other, string $description = '', bool $returnResult = false): ?bool
```

Comment:
this is the only difference with `ConstraintTraitForV7`
Can't we `use ConstraintTraitForV7;` and override only this method?

Reply:
We could, but I would personally keep it simple and leave it as it is.

Reply:
I could use `ConstraintTraitForV8`, but considering that it would be the only exception I opted against it.